oxidation/client/src/views/HomePage.vue

```diff
@@ -103,8 +103,8 @@
                               >
                                 <p>{{ $t('HomePage.organizationList.lastLogin') }}</p>
                                 <p>
-                                  {{ deviceStoredDataDict[device.slug] && 'lastLogin' in deviceStoredDataDict[device.slug] ?
-                                    formatLastLogin(deviceStoredDataDict[device.slug].lastLogin) : '--' }}
+                                  {{ device.slug in deviceStoredDataDict ?
+                                    $filters.formatTimeSince(deviceStoredDataDict[device.slug].lastLogin, '--') : '--' }}
```

Comment:
In this case if deviceStoredDataDict[device.slug] has no 'lastLogin', 'formatTimeSince' will receive undefined as parameter.
In the main.ts 'formatTimeSince' expect only a Date, not Date | undefined.
You can:
- change the type expected in main.ts
or
- restore the `'lastLogin' in deviceStoredDataDict[device.slug]` check in the condition

Reply:
My bad, the function in `date.ts` does allow undefined, just mistyped the function in `main.ts`.